### ISSUE TITLE
Error loading forms containing countdown-widget

### ISSUE THREAD

**Issue by jkuester** [medic/cht-core#7918]
**Describe the bug**
Forms using the [`countdown-widget` timer](https://github.com/medic/cht-core/blob/814ed5cd488d783784959e60c4066a665319accb/webapp/src/js/enketo/widgets/countdown-widget.js) fail to load in `4.x`.  

**To Reproduce**
1. Use a `4.x` version of cht-core
2. Upload a form that contains a countdown timer. (e.g. [this one](https://github.com/medic/support-scripts/blob/master/forms/countdown-timer.xml))
3. Try to open the form 

**Expected behavior**
The form and the timer widget should function without issue.

**Details**
This issue was unintentionally caused by [eslint refactoring changes](https://github.com/medic/cht-core/pull/7728/files#diff-9a33bc7a9218eeeda86158feb834f8987bec3945173989f44aa87e447f580970).  Basically, `loadSound` was changed from a `function` to a `const`. The problem is that `loadSound()` is being called on line `67` which is _before_ it is actually declared on line `70`.  Thanks to the "magic" of [function hoisting](http://www.adequatelygood.com/JavaScript-Scoping-and-Hoisting.html) in JS, you can apparently reference functions before they are declared but you cannot do the same for `const`....

The fix there should be simple, though.  Just need to move the declaration of `loadSound` up before it is used.  We should also add some kind of automated e2e test that will verify the behavior of this widget so this does not happen again!

**Logs**
Browser console logs:

```
feedback.service.ts:77 TypeError: loadSound is not a function
    at countdown-widget.js:67:16
    at new TimerAnimation (countdown-widget.js:60:18)
    at Timerwidget._init (countdown-widget.js:41:5)
    at new Widget (widget.js:21:21)
    at Timerwidget._createSuperInternal (createSuper.js:15:1)
    at new Timerwidget (countdown-widget.js:30:7)
    at Collection._instantiateSingleWidget (widgets-controller.js:203:19)
    at widgets-controller.js:227:43
    at Array.forEach (<anonymous>)
    at Collection.instantiate (widgets-controller.js:227:23)
```

**Environment**
- Instance: local
- Browser: Chromium
- Client platform: Linux
- App: webapp
- Version: 4.x (master)

**Additional context**
First noticed by @lorerod!

**Comment by jkuester** [comment]
One additional detail worth noting is that I don't think this issue affects anyone using the Android app.  It should be limited to the browser.

**Comment by garethbowen** [comment]
Ready for AT: #7567

**Comment by lorerod** [comment]
I'm a bit confused; the fix is not on [#7919 ](https://github.com/medic/cht-core/pull/7919)?

**Comment by garethbowen** [comment]
@lorerod Sorry! Yes, I've updated my comment with the right link!

**Comment by lorerod** [comment]
**Reproduced on 4.0**
**Environment**
Instance: Local using [cht-core.yml](https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3A4.0.0/docker-compose%2Fcht-core.yml) and [cht-couchdb.yml ](https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3A4.0.0/docker-compose%2Fcht-couchdb.yml)
Browser: Chrome
Client platform: MacOS
Version: 4.0.0-alpha

**App:** webapp
**Steps to reproduce:**
- Got my local instance up using compose files and `COUCHDB_PASSWORD=password COUCHDB_USER=admin docker-compose -f cht-couchdb.yml -f cht-core.yml up`
- On config-brac directory execute:
`cht --url=https://admin:password@192-168-100-62.my.local-ip.co:443 --accept-self-signed-certs --skip-validate`
- Login as admin and create places and offline user
- Login as offline user and created a Family, mother and baby
- For the baby try to open the Assess Patient form and got this error:
![Captura de Pantalla 2022-11-23 a la(s) 12 07 45](https://user-images.githubusercontent.com/21312057/203582507-5c7ee052-4b76-4182-85f2-4e3483d10720.png)

**App:** mobile with Samsung Galaxy A01. Android version 10. WebView 107.0.5304.105
- Login as offline user
- For the baby I open the Assess Patient form without error, and was able to use the countdown-widget:


![Screenshot_20221123-121844](https://user-images.githubusercontent.com/21312057/203587768-e54cfcfd-ca20-413f-9424-ddb7984e42b9.jpg)
![Screenshot_20221123-123454](https://user-images.githubusercontent.com/21312057/203587773-aefc2007-55a7-4987-a054-a802c55e9b00.jpg)
![Screenshot_20221123-123505](https://user-images.githubusercontent.com/21312057/203587774-930c5aab-f364-4318-aded-23c0550fdfd6.jpg)

**Comment by lorerod** [comment]
**Fixed on`7918-countdown-timer`**
**Environment**
Instance: Local using Development setup with branch `7918-countdown-timer`
Browser: Chrome
Client platform: MacOS
Version: `7918-countdown-timer`

App: webapp
**Steps to reproduce:**
- Got my local instance up 
- On config-brac directory execute:
`cht --url=https://admin:password@192-168-100-62.my.local-ip.co:443 --accept-self-signed-certs --skip-validate`
- Login as admin and create places and offline user
- Login as offline user and created a Family, mother and baby
- For the baby I open the Assess Patient form without error, and was able to use the countdown-widget:
![Captura de Pantalla 2022-11-23 a la(s) 13 06 58](https://user-images.githubusercontent.com/21312057/203594453-ecf57e96-36fa-4edb-9c30-cd20fb1ff1f7.png)
![Captura de Pantalla 2022-11-23 a la(s) 13 07 19](https://user-images.githubusercontent.com/21312057/203594505-ba456ad6-e99e-498d-ab03-165a026aa5e8.png)
![Captura de Pantalla 2022-11-23 a la(s) 13 07 13](https://user-images.githubusercontent.com/21312057/203594530-71e41a85-70d0-4a06-b507-ab3d66379d34.png)


App: mobile with Android
- Login as offline user
- For the baby I open the Assess Patient form without error, and was able to use the countdown-widget:
![Screenshot_20221123-133639](https://user-images.githubusercontent.com/21312057/203600873-1a78b73f-d96f-44d7-b2f5-7fa57e318ba8.jpg)
![Screenshot_20221123-133636](https://user-images.githubusercontent.com/21312057/203600880-ae2d381a-71b2-486c-826e-54003f24fe60.jpg)
![Screenshot_20221123-133622](https://user-images.githubusercontent.com/21312057/203600883-ba38eb3a-b417-419e-a014-6d2c344a5091.jpg)

**Comment by lorerod** [comment]
@jkuester @garethbowen this look good! thanks!

**Comment by garethbowen** [comment]
Merged and cherry picked to `4.0.x`.
